app/models/run.rb

```diff
@@ -36,11 +37,14 @@ class Run < ApplicationRecord
 
   class << self
     def programs
-      [LlanfairGered, Llanfair, Urn, LiveSplit, SplitterZ, TimeSplitTracker, WSplit]
+      [
+        ShitSplit, Splitty, Llanfair2, SplitterZ, FaceSplit, Llanfair2, Portal2LiveTimer,
+        LlanfairGered, Llanfair, Urn, LiveSplit, SplitterZ, TimeSplitTracker, WSplit
+      ]
```

Comment:
1. Llanfair2 and SplitterZ are in this list twice
2. Be sure to modify the top of the README, which specifies what timers we accept, and the v4 API doc, which specifies which timers we can output when given `Accept: application/original-timer`

Reply:
The output timers haven't changed AFAIK because core can only save as .lss and not any other formats.  So we would still need to write templates for any of those timers if we wanted to support outputting to them.

Reply:
i believe i understand that part, but we still can potentially return those formats if the original upload was in one of those unexportable formats, right?